THIRD-PARTY-LICENSES.properties

```diff
@@ -0,0 +1,72 @@
+# Generated by org.codehaus.mojo.license.AddThirdPartyMojo
+#-------------------------------------------------------------------------------
+# Already used licenses in project :
+# - Apache 2
+# - Apache 2.0
+# - Apache 2.0 License
+# - Apache License 2.0
+# - Apache License, Version 2.0
+# - Apache License, version 2.0
+# - Apache Software License - Version 2.0
+# - Apache Software Licenses
+# - Apache-2.0
+# - BSD 3-clause
+# - BSD New
+# - BSD New license
+# - BSD-2-Clause
+# - CC0
+# - CDDL + GPLv2 with classpath exception
+# - CDDL 1.1
+# - CDDL+GPL License
+# - CDDL/GPLv2+CE
+# - Common Public License Version 1.0
+# - EDL 1.0
+# - Eclipse Distribution License - v 1.0
+# - Eclipse Public License - Version 1.0
+# - Eclipse Public License - v 1.0
+# - Eclipse Public License 1.0
+# - GNU Lesser General Public License
+# - GPL2 w/ CPE
+# - LGPL 2.1
+# - MIT
+# - MIT License
+# - MIT license
+# - MPL 1.1
+# - New BSD License
+# - The Apache License, Version 2.0
+# - The Apache Software License, Version 2.0
+# - The MIT License
+# - Three-clause BSD-style
+#-------------------------------------------------------------------------------
+# If there is a missing license file the command mvn clean install will provide a
+# warning listing the artifact with the missing license. The artifact and the
```

Comment:
In a conversation on this PR, didn't you say it would be an error? At least in terms of the build failing?

Reply:
Fixed comment